src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -429,4 +440,28 @@ trait WorkflowComponent {
 
   }
 
+  object workflowAuditStatusQuery extends TableQuery(new WorkflowAuditStatusTable(_)) {
+    type workflowAuditStatusQueryType = Query[WorkflowAuditStatusTable, WorkflowAuditStatusRecord, Seq]
+
+    def queueTimeMostRecentSubmittedWorkflow: ReadAction[Long] = {
+      // for the most-recently submitted workflow, find the time it was submitted and the earliest time we have recorded.
+      // we could query specifically for the time this workflow was Queued, but we'll just use the earliest time
+      // for resiliency, in case the workflow somehow skipped Queued status.
+      val timingQuery = for {
+        submitted <- workflowAuditStatusQuery.filter(_.status === WorkflowStatuses.Submitted.toString).sortBy(_.timestamp.desc).take(1)
```

Comment:
head is illegal syntax here. You can use head after getting a result; I think you have to use take to get `LIMIT 1` into the sql statement.